sys-process/fcron/fcron-3.3.1-r2.ebuild

```diff
@@ -0,0 +1,273 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOMAKE="none"
+
+inherit autotools cron flag-o-matic pam systemd user-info
+
+MY_PV="${PV/_beta/}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="A command scheduler with extended capabilities over cron and anacron"
+HOMEPAGE="http://fcron.free.fr/"
+SRC_URI="http://fcron.free.fr/archives/${MY_P}.src.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="audit debug pam selinux l10n_fr +mta +system-crontab readline"
+
+DEPEND="
+	acct-group/fcron
+	acct-user/fcron
+	virtual/libcrypt:=
+	audit? ( sys-process/audit )
+	pam? ( sys-libs/pam )
+	readline? ( sys-libs/readline:= )
+	selinux? ( sys-libs/libselinux )
+"
+
+RDEPEND="
+	${DEPEND}
+	app-misc/editor-wrapper
+	mta? ( virtual/mta )
+	pam? ( sys-auth/pambase )
+"
+
+PATCHES=(
+	"${FILESDIR}"/fcron-3.1.1-noreadline.patch
+	"${FILESDIR}"/fcron-3.2.1-musl-getopt-order.patch
+	"${FILESDIR}"/fcron-3.3.1-gcc15-bool.patch
+)
+
+pkg_setup() {
+	rootuser=$(egetent passwd 0 | cut -d ':' -f 1)
+	[[ ${rootuser} ]] || rootuser=root
+	rootgroup=$(egetent group 0 | cut -d ':' -f 1)
+	[[ ${rootgroup} ]] || rootgroup=root
+}
+
+src_prepare() {
+	default
+
+	# respect LDFLAGS
+	sed "s:\(@LIBS@\):\$(LDFLAGS) \1:" -i Makefile.in || die "sed failed"
+
+	# Adjust fcrontab path
+	sed -e 's:/etc/fcrontab:/etc/fcron/fcrontab:' -i script/check_system_crontabs.sh || die
+
+	mv configure.in configure.ac || die
+
+	cp "${FILESDIR}"/crontab.2 "${WORKDIR}"/crontab || die
+
+	eautoconf
+}
+
+src_configure() {
+	# Don't try to pass --with-debug as it'll play with cflags as
+	# well, and run foreground which is a _very_ nasty idea for
+	# Gentoo.
+	use debug && append-cppflags -DDEBUG
+
+	# bindir is used just for calling fcronsighup
+	local myeconfargs=(
+		--with-cflags="${CFLAGS}"
+		--bindir=/usr/libexec
+		--sbindir=/usr/libexec
+		$(use_with audit)
+		$(use_with mta sendmail)
+		$(use_with pam)
+		$(use_with readline)
+		$(use_with selinux)
+		--sysconfdir=/etc/fcron
+		--with-username=fcron
+		--with-groupname=fcron
+		--with-piddir=/run
+		--with-spooldir=/var/spool/fcron
+		--with-fifodir=/run
+		--with-fcrondyn=yes
+		--disable-checks
+		--with-editor=/usr/libexec/editor
+		--with-shell=/bin/sh
+		--without-db2man
+		--without-dsssl-dir
+		--with-rootname=${rootuser}
+		--with-rootgroup=${rootgroup}
+		--with-systemdsystemunitdir="$(systemd_get_systemunitdir)"
+	)
+
+	econf "${myeconfargs[@]}"
+}
+
+src_compile() {
+	default
+
+	# bug #216460
+	sed \
+		-e 's:/usr/local/etc/fcron:/etc/fcron/fcron:g' \
+		-e 's:/usr/local/etc:/etc:g' \
+		-e 's:/usr/local/:/usr/:g' \
+		-i doc/*/*/*.{txt,1,5,8,html} \
+		|| die "unable to fix documentation references"
+}
+
+src_install() {
+	emake install DESTDIR="${DISTDIR}" BOOTINSTALL=0
+	return
```

Comment:
What's this about?

Reply:
oops, that was me testing something, and forgot to revert those changes

Reply:
Fixed.

I was investigating how other distros deal with https://bugs.gentoo.org/944230 - and they call make install instead of what this ebulid does, but the answer is that they don't deal with it, and the permissions are exactly same - fcron-writeable

Reply:
eh, I meant https://bugs.gentoo.org/925512